shared/util/forward-logs.desktop.js

```diff
@@ -97,6 +97,7 @@ function setupTarget () {
       }
     }
 
+    // $FlowIssue these can no longer be written to
```

Comment:
maybe we should move towards just using a logger interface and entirely ditch console.log